tests/src/Integration/Action/SystemMailToUsersOfRoleTest.php

```diff
@@ -0,0 +1,238 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\rules\Integration\Action\SystemMailToUsersOfRoleTest.
+ */
+
+namespace Drupal\Tests\rules\Integration\Action;
+
+use Drupal\Tests\rules\Integration\RulesEntityIntegrationTestBase;
+use Drupal\Core\Language\LanguageInterface;
+use Drupal\Component\Utility\SafeMarkup;
+use Drupal\Tests\rules\Integration\RulesUserIntegrationTestTrait;
+
+/**
+ * @coversDefaultClass \Drupal\rules\Plugin\RulesAction\SystemMailToUsersOfRole
+ * @group rules_actions
+ */
+class SystemMailToUsersOfRoleTest extends RulesEntityIntegrationTestBase {
+
+  use RulesUserIntegrationTestTrait;
+
+  /**
+   * @var \Psr\Log\LoggerInterface
+   */
+  protected $logger;
+
+  /**
+   * @var \Drupal\Core\Mail\MailManagerInterface
+   */
+  protected $mailManager;
+
+  /**
+   * @var \Drupal\user\UserStorage
+   */
+  protected $userStorage;
+
+  /**
+   * @var \Drupal\user\Entity\Role[]
+   */
+  protected $roles;
+
+  /**
+   * @var \Drupal\user\Entity\User[]
+   */
+  protected $users;
+
+  /**
+   * The action to be tested.
+   *
+   * @var \Drupal\rules\Plugin\RulesAction\SystemMailToUsersOfRole
+   */
+  protected $action;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() {
+    parent::setUp();
+    $this->enableModule('user');
+    $this->namespaces[] = '';
+
+    $this->logger = $this->getMock('Psr\Log\LoggerInterface');
```

Comment:
Since Drupal now requires PHP 5.5 we can use LoggerInterface::class here instead of a raw string. Also elsewhere.